dev/InfoBar/TestUI/Themes/InfoBarStyles.xaml

```diff
@@ -0,0 +1,244 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp">
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="InfoBarSeverityCriticalStatusColor" Color="#FDE7E9"/>
+            <SolidColorBrush x:Key="InfoBarSeverityWarningStatusColor" Color="#FFF4CE"/>
+            <SolidColorBrush x:Key="InfoBarSeveritySuccessStatusColor" Color="#DFF6DD"/>
+            <SolidColorBrush x:Key="InfoBarSeverityDefaultStatusColor" Color="#F2F2F2"/>
+            <SolidColorBrush x:Key="InfoBarHyperlinkButtonColor" Color="Black"/>
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="Dark">
+            <SolidColorBrush x:Key="InfoBarSeverityCriticalStatusColor" Color="#442726"/>
+            <SolidColorBrush x:Key="InfoBarSeverityWarningStatusColor" Color="#433519"/>
+            <SolidColorBrush x:Key="InfoBarSeveritySuccessStatusColor" Color="#393D1B"/>
+            <SolidColorBrush x:Key="InfoBarSeverityDefaultStatusColor" Color="#2b2b2b"/>
+            <SolidColorBrush x:Key="InfoBarHyperlinkButtonColor" Color="White"/>
+        </ResourceDictionary>
+    </ResourceDictionary.ThemeDictionaries>
+
+    <x:Double x:Key="InfoBarFontSize">14</x:Double>
+
+    <x:Double x:Key="InfoBarMinHeight">40</x:Double>
+
+    <x:Double x:Key="InfoBarCloseButtonSize">44</x:Double>
+    <x:Double x:Key="InfoBarCloseButtonGlyphSize">12</x:Double>
+    
+    <Style TargetType="local:InfoBar" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:InfoBar">
+                    <Border x:Name="Container" 
+                            VerticalAlignment = "Top"
+                            Background = "Transparent"                            
+                            AutomationProperties.Name="{TemplateBinding AutomationProperties.Name}">
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="ButtonStates">
+                                <VisualState x:Name="NoButtonsVisible">
```

Comment:
Just curious, should there be a state with the HyperlinkButton?

Reply:
I've just been treating my HyperlinkButton like I've been treating title and message, so if HyperlinkButton content is an empty string or null, then the element just simply doesn't show, and I modify the margins dynamically in InfoBar.cs. Functionally, it works, but maybe a visual state is better stylistically??

Reply:
Just added a visual state for the hyperlinkbutton to collapse and make visible with my most recent push.